docs/supported-mutators.md

```diff
@@ -293,32 +296,34 @@ Strings and literals identified to a regex are mutated in the following way:
 
 ## String Literal
 
-| Original                               | Mutated               |
-| -------------------------------------- | --------------------- |
-| `"foo"` (non-empty string)             | `""` (empty string)   |
-| `""` (empty string)                    | `"Stryker was here!"` |
-| `s"foo ${bar}"` (string interpolation) | `s""` ¹               |
+| Mutant operator                            | Original                               | Mutated               |
+| ------------------------------------------ | -------------------------------------- | --------------------- |
+| FilledStringLiteralToEmptyReplacement      | `"foo"` (filled string)                | `""` (empty string)   |
+| EmptyStringLiteralToFilledReplacement      | `""` (empty string)                    | `"Stryker was here!"` |
+| FilledInterpolatedStringToEmptyReplacement | `s"foo ${bar}"` (string interpolation) | `s""` ¹               |
```

Comment:
Minor thing, but as far as I'm concerned, this can also be `FilledStringLiteralToEmptyReplacement`. I don't think Stryker-JS, for example, has a difference between `""` and ``